blocks/related-blogs/related-blogs.css

```diff
@@ -10,3 +10,128 @@
     color: #97f5f7;
     margin-bottom: 25px;
 }
+
+.related-list-item {
```

Comment:
All the CSS definitions should be scoped with the block `.related-blogs`

Reply:
done